src/data/unicorns.json

```
@@ -121,4 +121,18 @@
     "color": "#7C4DFF",
     "roles": ["developer"]
   }
```

```suggestion
  },
```

We need a comma here, that's why the build is breaking